schema.sql

```diff
@@ -10,3 +10,35 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals ADD COLUMN species VARCHAR(100);
+
+/* Schema for owners table */
+
+CREATE TABLE owners (
+  id SERIAL NOT NULL PRIMARY KEY,
+  full_name VARCHAR(255) NOT NULL,
+  age INTEGER NOT NULL
+);
+
+/* Schema for species table */
+
+CREATE TABLE species (
+  id SERIAL NOT NULL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL
+);
+
+/* modify animal table */
+
+ALTER TABLE animals
+ALTER COLUMN id SET DATA TYPE SERIAL PRIMARY KEY;
```

Comment:
- [ ] Kindly make sure to drop/remove the column `species` from the animal table

Reply:
done